packages/storybook/stories/va-back-to-top.stories.jsx

```diff
@@ -25,7 +25,7 @@ const Template = ({
       <div className="usa-grid usa-grid-full">
         <div className="usa-width-three-fourths">
           <h1>The top</h1>
-          {[...Array(displayAmount)].map(i => (
+          {[...Array(displayAmount)].map((_, i) => (
```

Comment:
Fix for "keys must be unique error". `[...Array(displayAmount)]` generates an empty array where each 'value' of the array is "undefined", so this fix skips the value of the array and instead uses the array's index as the key.